packages/cli/src/zosuss/ZosUss.definition.ts

```diff
@@ -15,7 +15,7 @@ import { SshSession } from "@zowe/zos-uss-for-zowe-sdk";
 
 export const definition: ICommandDefinition = {
     name: "zos-ssh",
-    aliases: ["ssh", "zos-uss", "uss"],
+    aliases: ["ssh", "zos-uss", "uss", "uf"],
```

Comment:
Is this appropriate for the zos-ssh and zos-uss group? `uf` tends to imply a uss file, which this group does not manipulate (except maybe through ssh commands), and is more appropriate for the `zos-files` group.